testplan/common/report/base.py

```diff
@@ -201,6 +201,14 @@ def flattened_entries(self, depth):
         """
         return [(depth, entry) for entry in self]
 
+    @property
+    def hash(self):
```

Comment:
Read the docstring below - __hash__ should only be used for immutable objects.

Reply:
For the internet's future reference: [link](https://docs.python.org/3/reference/datamodel.html#object.__hash__)

"If a class defines mutable objects and implements an __eq__() method, it should not implement __hash__(), since the implementation of hashable collections requires that a key’s hash value is immutable (if the object’s hash value changes, it will be in the wrong hash bucket)."